### ISSUE TITLE
Finetuning for downstream segmentation task 

### ISSUE THREAD

**Issue by agraham9966** [yformer/EfficientSAM#32]
Hi there - I noticed the new notebooks added for segment anything example. I am wondering if you can give any advice for how to finetune it on a custom dataset for semantic segmentation? Will there be another notebook released which will cover this? 

Thanks

**Comment by balakv504** [comment]
Great question!
We can try to provide a code for finetuning once we finish address some of the ongoing issues (providing examples for multiple objects, onnx export, etc).
Few questions: How many images are there in your custom dataset? If it is small (O(few thousand)),  you can try to instantiate the model (as is done in the example notebook) to start the FT process.

**Comment by yformer** [comment]
@agraham9966, Yes, we added the segment anything example. Do you mean finetuning it for segment anything as SAM, or semantic anything as mask2former?
